### PR TITLE
untracked_devices: Untrack lynx

### PR DESCRIPTION
**Pull request by hellobbn**

### CODE REVIEW

**Review by MartinX3 — APPROVED** [review]

File: untracked_devices.xml [hunk 1]
```diff
@@ -35,6 +35,7 @@
     <!-- Device repo provides core libraries for ie. microdroid -->
     <!-- <remove-project name="device/google/cuttlefish" /> -->
     <!-- <remove-project name="device/google/cuttlefish_prebuilts" /> -->
+    <remove-project name="device/google/lynx" />
     <remove-project name="device/google/gs201" />
     <remove-project name="device/google/gs201-sepolicy" />
     <remove-project name="device/google/pantah" />
```
Comment (MarijnS95) [inline untracked_devices.xml:41]: How about we keep this sorted alphabetically? The rest is, afaik.

**Review by MarijnS95 — COMMENTED** [review]
No commit description again? No other repos to untrack?

**Comment by MarijnS95** [comment]
And please add `Fixes: https://github.com/sonyxperiadev/bug_tracker/issues/800` to the PR description.

**Comment by hellobbn** [comment]
> And please add `Fixes: https://github.com/sonyxperiadev/bug_tracker/issues/800` to the PR description.

Should I add this to commit description or the description of this pull request?

**Comment by hellobbn** [comment]
I have added description. There is no other repo to untrack to allow building AOSP.
I have added the `Fixes` line to the commit description

**Comment by MarijnS95** [comment]
The `Fixes` line is better in the PR, but it works in the commit description as well.

**Comment by MartinX3** [comment]
So the PR can get merged?

**Review by jerpelea — APPROVED** [review]

**Pull request merged**